docs/synchronization.md

```diff
@@ -147,8 +152,29 @@ Examples:
 1. [Step level semaphore](https://github.com/argoproj/argo-workflows/blob/main/examples/synchronization-tmpl-level.yaml)
 1. [Step level mutex](https://github.com/argoproj/argo-workflows/blob/main/examples/synchronization-mutex-tmpl-level.yaml)
 
-### Other Parallelism support
+## Queuing
+
+When a Workflow cannot acquire a lock it will be placed into a ordered queue.
+
+You can set a [`priority`](parallelism.md#priority) on Workflows.
+The queue is first ordered by priority: a higher priority number is placed before a lower priority number.
+The queue is then ordered by `creationTimestamp`: older Workflows are placed before newer workflows.
```

Comment:
```suggestion
When a workflow cannot acquire a lock it will be placed into a ordered queue.

You can set a [`priority`](parallelism.md#priority) on workflows.
The queue is first ordered by priority: a higher priority number is placed before a lower priority number.
The queue is then ordered by `creationTimestamp`: older workflows are placed before newer workflows.
```

lower-case "workflows" for consistency